CRM/Civisocial/Page/Login.php

```diff
@@ -24,7 +20,7 @@ public function run() {
 
     if (count($path) == 3 && $path[2] == 'logout') {
       $oap->login();
```

Comment:
while this will act as logout, but ideally you should use a separate method to clean session in case of logout. Currently this will just set `civisocial_logged_in` as false. But you should also remove the rest of the variables from session.